releasenotes/notes/remove-wire-cutting-ad0e7ab8af699561.yaml

```
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    Removed the :mod:`circuit_knitting.wire_cutting` module. Users are now expected to use the :mod:`circuit_knitting.cutting.cutqc` module instead.
```

I think this is perhaps more clear about the fact that no code changes are required; it's only the import location that changed.

```suggestion
    Removed the :mod:`circuit_knitting.wire_cutting` module. Users are now expected to import from the :mod:`circuit_knitting.cutting.cutqc` module instead.
```